nrf_modem/doc/ug_nrf_modem_porting_os.rst

```diff
@@ -53,27 +53,31 @@ nrf_modem_os_timedwait
 This function is called by the Modem library when a timed context or sleep is required.
 A blind return value of 0 will make all the Modem library operations always blocking.
 
-Note: Calls to :c:func:`nrf_modem_os_timedwait` must be able to enter a waiting state regardless of the modem library and modem OS initialization status. Hence, it must be possible to call :c:func:`nrf_modem_os_timedwait` before any call to :c:func:`nrf_modem_os_init`. To ensure that all waiting threads are woken on :c:func:`nrf_modem_event_notify`, the waiting threads must not be cleared on :c:func:`nrf_modem_os_init` or :c:func:`nrf_modem_os_shutdown`.
+.. note::
+   The calls to :c:func:`nrf_modem_os_timedwait` function must be able to enter a waiting state regardless of the modem library and modem OS initialization status.
+   Hence, it must be possible to call the :c:func:`nrf_modem_os_timedwait` function before any call to the :c:func:`nrf_modem_os_init` function.
```

Comment:
This is no longer the case, right @eivindj-nordic?

Reply:
true

Reply:
thanks for clarifying

Reply:
Removed text